tests/test_create_post_page.py

```diff
@@ -0,0 +1,103 @@
+import logging
+
+import pytest
+from selenium import webdriver
+
+from constants.base import DRIVER_PATH, BASE_URL
+from pages.start_page import StartPage
+from pages.utils import random_str
+
+
+class TestCreatePostPage:
+    log = logging.getLogger("[CreatePostPage]")
+
+    @pytest.fixture(scope="function")
+    def open_start_page(self):
+        """Open Start page"""
+        # create driver
+        driver = webdriver.Chrome(DRIVER_PATH)
+        # open Start Page URL
+        driver.get(BASE_URL)
+        driver.implicitly_wait(1)
+        # Steps
+        yield StartPage(driver)
+        # Close driver
+        driver.close()
+
+    @pytest.fixture(scope="function")
+    def open_hello_page(self, open_start_page):
+        """Open Hello page"""
+        # username_value = f"{random_str()}{random_num()}"
+        # email_value = f"{random_str()}{random_num()}@mail.com"
+        # password_value = f"{random_str(6)}{random_num()}"
+        # return open_start_page.sign_up_and_verify(username_value, email_value, password_value)
+        username_value = "nicewarthog"
+        password_value = "nicewarthogpass"
+        open_start_page.header.sign_in_and_verify(username_value, password_value)
+        self.log.info("Logged in with correct login and password")
+        return open_start_page.header.hello_page_return()
+
+    @pytest.fixture(scope="function")
+    def open_create_post_page(self, open_hello_page):
+        create_post_page = open_hello_page.header.navigate_to_create_post()
+        self.log.info("Move to Create Post Page")
+        return create_post_page
+
+    @pytest.fixture(scope="function")
+    def create_new_post(self, open_create_post_page):
+        new_post = open_create_post_page.create_post(title=random_str(15), body=random_str(50))
+        return new_post
+
+    def test_create_new_post(self, create_new_post, open_create_post_page):
+        """
+        Pre-conditions:
+        - Sign Up/Sign In as the user
+        - Navigate to Create Post Page
+        - Create new post
```

Comment:
Sounds weird. So if you cannot create post that wouldn't be a problem of this test ?